### PR TITLE
development tag should be in monofont

### PR DESCRIPTION
**Pull request by esc**
As title

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/first_rc_checklist.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ labels: task
 ### Post Release:
 
 * [ ] Clean up `ci_artifacts` by moving files to sub-directories
-* [ ] Tag X.Y+1.0dev0 to start new development cycle on `main`.
+* [ ] Tag `X.Y+1.0dev0` to start new development cycle on `main`.
```
Comment (stuartarchibald) [inline .github/ISSUE_TEMPLATE/first_rc_checklist.md:37]: Fix the `Annotated tag X.Y.Zrc1` line above too?

Comment (esc) [reply .github/ISSUE_TEMPLATE/first_rc_checklist.md:37]: yes, added in c622c0b774ecf0c63cf4eec9176924e02d52bb50

**Review by stuartarchibald — DISMISSED** [review]
Thanks for the patch and fixes.

**Comment by esc** [comment]
rebased to fix conflict

**Review by stuartarchibald — APPROVED** [review]

**Pull request merged**